cli/src/commands/lock.ts

```diff
@@ -0,0 +1,42 @@
+import { CLIError } from '@oclif/errors';
+import Command from '../utils/command';
+import flags from '../utils/cli-flags';
+
+export default class Lock extends Command {
+  public static description = 'lock an application';
+  public static hidden = true;
+  public static examples = [
+    `$ ${Command.cliBinName} lock --reason template application`,
+    `$ ${Command.cliBinName} lock --reason application is a template great-unicorn-42`,
```

Comment:
I don't think this command works, because of shell quoting issues.  AFAIU you would need to put quotes around the argument after `--reason`.

Reply:
I don't understand.  How is this different from https://github.com/reshufflehq/reshuffle/pull/357/files#diff-fde2f025e0c0cb2bbf4e62bee59cdd61R47?  That *is* a test that you ran, and it *does* find an error message https://github.com/reshufflehq/reshuffle/pull/357/files#diff-a69e11f3345230f4180fe8f079b287f5R25.  Could you please specify here what tests you ran?